widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -285,7 +286,12 @@ export function ConfirmWalletsModal(props: PropTypes) {
             <Typography variant="headline" size="small">
               {i18n.t({
                 id: 'Your {blockchainName} wallets',
-                values: { blockchainName: showMoreWalletFor },
+                values: {
+                  blockchainName: getBlockchainShortNameFor(
```

Comment:
It it possible to also update blockchain name here? :bow: 

![Screenshot from 2023-11-21 15-48-28](https://github.com/rango-exchange/rango-client/assets/99180934/deeb11e0-ba28-4ec5-bd7a-0ef8f30d2d63)